contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -102,12 +145,35 @@ linked_server_err_handler(LinkedServerProcess lsproc, int severity, int db_error
 {
 	StringInfoData buf;
 
+	char* err_msg = NULL;
+	char* str =  NULL;
+	char* pos = NULL;
+
 	initStringInfo(&buf);
 
+	/*
+	 * We remove "Adaptive" from error message since we are only
+	 * supporting remote servers that use T-SQL and communicate over TDS
+	 */
+	err_msg = remove_substr(pstrdup(db_err_str), "Adaptive ");
+	str = err_msg;
+
+	/* "Server" --> "server" */
```

Comment:
Bit more details would be helpful to improve the readability

Reply:
done